view/theme/frio/css/style.css

```diff
@@ -1908,6 +1937,7 @@ wall-item-comment-wrapper.well hr {
 
 .comment-edit-submit-wrapper {
     text-align: right;
+    /* margin-bottom: 0; */
```

Comment:
Commenting is fine for development, but please don't commit them.
```suggestion
```

Reply:
This actually need to be uncommented :/

Reply:
I checked and I see it uncommented now in the "Files changed". No idea why.

I want this to remove the large gap between the Submit/preview botton and the end of the panel when commenting.

Reply:
I may have commented on a subset of the PR's commit, sorry about the noise.